hnn_core/tests/test_parallel_backends.py

```diff
@@ -182,7 +182,7 @@ def test_compare_hnn_core(self, run_hnn_core_fixture, backend, n_jobs=1):
                                      'L2_pyramidal': 113,
                                      'L5_pyramidal': 395,
                                      'L5_basket': 85,
-                                     'evdist1': 234,
+                                     'evdist1': 269,
                                      'evprox2': 269}
```

Comment:
Does it seem odd to anyone that a proximal drive creates 269 spikes instead of the 270 I'd expect for each cell in the network?

Reply:
perhaps a question for @ntolley . I don't know where these numbers came from

Reply:
haha, spotted the bug. The count starts from 0 above instead of 1. See L174

Reply:
Duh. Wow, I remember looking over that line too but totally missed the problem. Fixed in https://github.com/jonescompneurolab/hnn-core/pull/383/commits/9bc63a4d31a387d9dfbc19d7e48e7fb2e34052c0. Thanks @jasmainak!